pkg/runners/zero/zero_test.go

```diff
@@ -186,7 +186,7 @@ func TestBitwiseBuiltin(t *testing.T) {
         [ap + 1] = 9;
         [ap + 2] = 15;
         ret;
-    `, sn.Bitwise)
+    `, "starknet_with_keccak", sn.Bitwise)
```

Comment:
yea thats right. This was done not by me, so I'm not sure what was the idea. But yea I agree it could be moved somewhere else, for instance to the builtins package. The `UnmarshalJSON` is not used at all 😛